src/components/ItemDetail/Cite.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MultiMetadata from './MultiMetadata';
+import SingleMetadata from './SingleMetadata';
+
+const Cite = props => {
+  if (!props.item) {
+    return [];
+  }
+
+  const {
+    title = null,
+    permalink = null,
+    identifier = null,
+    license = null,
+    use_statement = null
+  } = props.item;
+
+  let formatMLA = () => {
+    return [title.primary + " here's the rest MLA"];
+  };
```

Comment:
For a single value calculation, I'd keep it a straight value rather than an `array` data type.   Also, since there's no real calculations occurring in the block, I'd refactor these from functions to a variable declaration.  ie.
```
let formatMLA = `${title.primary} here's the rest MLA`;
```
Note the backticks, this is a really handy ES6 feature for string concatenation.